### DISCUSSION TITLE
mvnDebug clean verify - Unable to insert breakpoint Absent Line Number Information

### DISCUSSION THREAD

**Discussion by vogella** [eclipse-tycho/tycho#3294] [Q&A]
Debugging a Tycho build (using the Oomph setup) I get the following popup:

![image](https://github.com/eclipse-tycho/tycho/assets/139910/d743fb46-fca2-49c1-84a5-2b853cd699ee)

Compiler options seem correct to me: 

![image](https://github.com/eclipse-tycho/tycho/assets/139910/d0bf985f-5af8-4731-b4a5-365c6fdde47f)

Any idea what this error is? It seems that after clicking this error away I still can debug the Tycho build.

**Comment by laeubi** [comment]
If you look at the message you will see that this is a dynamic proxy, not sure if JDT can do anything here I also noticed that in the past, interestingly if you just dispose the message the breakpoint still works.
